blenderproc/python/types/MeshObjectUtility.py

```diff
@@ -522,6 +523,20 @@ def add_modifier(self, name: str, **kwargs):
         for key, value in kwargs.items():
             setattr(modifier, key, value)
 
+    def mesh_as_trimesh(self) -> Trimesh:
```

Comment:
To avoid this duplicate function, you could just add a non-member function to BlenderUtility, e.g. 

```python
def mesh_as_trimesh(entity: Entity) -> Optional[trimesh]
   if isinstance(entity, Link):
   ...
   elif isinstance(entity, MeshObject):
   ...
   else:
   "Only meshobjects and Links are currently supported"
```

Reply:
I think that is not necessary as `self.visuals[0]` of `Link` is a `MeshObject`, so you could just replace the duplicate code in `Link` with:
```
        return self.visuals[0].mesh_as_trimesh()
```

Reply:
okay fine as well, just wanted to avoid having two functions that do the same :)

Reply:
true, thanks! (addressed https://github.com/DLR-RM/BlenderProc/pull/881/commits/691b4423b7feda9979e0baf1268225b6969fbdaa)